client/packages/requisitions/src/RequestRequisition/DetailView/DetailView.tsx

```diff
@@ -61,22 +63,25 @@ export const DetailView: FC = () => {
         )
       );
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     []
   );
 
+  const handleRowClick = (line: RequestLineFragment) => {
+    setSelectedItemId(line.item.id);
+    onOpen();
+  };
 
-  useEffect(() => {
-    setCustomBreadcrumbs({ 1: data?.requisitionNumber.toString() ?? '' });
-  }, [setCustomBreadcrumbs, data?.requisitionNumber]);
-
-  if (isLoading) return <DetailViewSkeleton />;
-
-  const onAddItem = () => {
-    navigate(buildItemEditRoute(data?.id, 'new'));
+  const handleAddItem = () => {
+    setSelectedItemId('new');
```

Comment:
'new' shouldn't be needed now that we aren't using it for the route - for our other modals, the pattern has been to use the `entity` from useEditModal, and open with a value for edit, or no value for new... example here:

https://github.com/msupply-foundation/open-msupply/compare/7398-internal-order-line-edit-modal...7398-internal-order-line-edit-modal(use-useEditModal-entity)